info.php

```diff
@@ -213,22 +212,22 @@ function get_mt_filter_enabled($existsfilter){
     return $filterenabled;
 }
 
-function get_exists_editor($editorname) {
+function get_editor_exists_and_enabled($editorname) {
```

Comment:
Maybe a clearer name ? Like `editor_exists_and_enabled`  if the return is `true` or `false`. In my opinion if the name contains get I assume it returns something else than a bool

Reply:
You’re right! But doing this would disrupt the all-get pattern... What does everyone think about that?

Reply:
I'd leave the name as is, I agree with Ubaldo the name could be improved but in this context is not that important, and keeping the pattern seems more important in this case.